app/service/ingest.py

```diff
@@ -93,17 +97,17 @@ def save_families(
 
 @validate_call(config=ConfigDict(arbitrary_types_allowed=True))
 def save_documents(
-    document_data: list[dict],
+    document_data: list[dict[str, Any]],
     corpus_import_id: str,
     db: Optional[Session] = None,
 ) -> list[str]:
     """
     Creates new documents with the values passed.
 
-    :param list[dict] document_data: The data to use for creating documents.
+    :param list[dict[str, Any]] document_data: The data to use for creating documents.
     :param str corpus_import_id: The import_id of the corpus the documents belong to.
-    :param Optional[Session] The database session to use for saving documents.
-    :return str: The new import_ids for the saved documents.
+    :param Optional[Session] The database session to use for saving documents or None.
```

Comment:
Missing parameter name.
```suggestion
    :param Optional[Session] db: The database session to use for saving documents or None.
```